clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/loadbalancer/UpsertGoogleHttpLoadBalancerAtomicOperation.groovy

```diff
@@ -239,7 +239,8 @@ class UpsertGoogleHttpLoadBalancerAtomicOperation extends UpsertGoogleLoadBalanc
         Boolean differentSessionCookieTtl = existingService.getAffinityCookieTtlSec() != backendService.affinityCookieTtlSec
         Boolean differentCDN = existingService.getEnableCDN() != backendService.enableCDN
         Boolean differentPortName = existingService.getPortName() != backendService.portName
-        if (differentHealthChecks || differentSessionAffinity || differentSessionCookieTtl || differentCDN || differentPortName) {
+        Boolean differentConnectionDraining = existingService.getConnectionDraining()?.getDrainingTimeoutSec() != description.backendService?.connectionDrainingTimeoutSec
```

Comment:
Its it just `backendService?.connectionDrainingTimeoutSec`? (Thought that HTTP load balancers had a list of backend services).

Reply:
This is in a `.each` loop.